lib/tasks/multitenant/tenants.rake

```diff
@@ -20,6 +20,13 @@ namespace :multitenant do
       puts(query.any? ? 'Some of the tenants haven\t been suspended' : 'All the right tenants have been suspended')
     end
 
+    desc 'Fix in the background tenant_id missing in alerts, log entries and backend apis'
+    task :fix_missing_in_background => :environment do |_task, relations|
```

Comment:
```suggestion
    task :fix_missing_tenant_id_async => :environment do |_task, relations|
```

because I'm an annoying person :upside_down_face: 
it's just 2 characters longer

Reply:
I would argue if I was going to use this more than once in my life. But so be it.